syntaxes/omt.tmLanguage.json

```diff
@@ -399,6 +399,17 @@
                     "contentName": "meta.embedded.block.odt",
                     "patterns": [{ "include": "source.odt" }]
                 },
+                {
```

Comment:
turn this into a separate item and include in both activity and procedure

Reply:
Done